rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -161,35 +161,33 @@ def rocalRun(self):
         """ Run the pipeline using rocalRun call
         """
         status = b.rocalRun(self._handle)
-        if(status != types.OK):
-            print("Rocal Run failed")
         return status
 
-    def define_graph(self):
+    def defineGraph(self):
         """This function is defined by the user to construct the
         graph of operations for their pipeline.
         It returns a list of outputs created by calling ROCAL Operators."""
-        print("definegraph is deprecated")
+        print("defineGraph is deprecated")
         raise NotImplementedError
 
-    def get_handle(self):
+    def getHandle(self):
         return self._handle
 
-    def GetOneHotEncodedLabels(self, array, device):
-        if device=="cpu":
-            if (isinstance(array,np.ndarray)):
+    def getOneHotEncodedLabels(self, array, device):
+        if device == "cpu":
+            if (isinstance(array, np.ndarray)):
                 b.getOneHotEncodedLabels(self._handle, array.ctypes.data_as(ctypes.c_void_p), self._numOfClasses, 0)
             else: #torch tensor
                 return b.getOneHotEncodedLabels(self._handle, ctypes.c_void_p(array.data_ptr()), self._numOfClasses, 0)
         else:
-            if (isinstance(array,cp.ndarray)):
+            if (isinstance(array, cp.ndarray)):
                 b.getCupyOneHotEncodedLabels(self._handle, array.data.ptr, self._numOfClasses, 1)
             else: #torch tensor
                 return b.getOneHotEncodedLabels(self._handle, ctypes.c_void_p(array.data_ptr()), self._numOfClasses, 1)
 
-    def set_outputs(self, *output_list):
+    def setOutputs(self, *output_list):
         self._output_list_length = len(output_list)
-        b.setOutputImages(self._handle,len(output_list),output_list)
+        b.setOutputImages(self._handle, len(output_list), output_list)
```

Comment:
Already we found length of output list and stored in L189
we can use self._output_list_length in L190 instead of recalculating again

Reply:
Or we could remove the line LINE 189 , if that varibale is not used?